src/pages/ProtocolPage.jsx

```diff
@@ -167,9 +170,11 @@ function ProtocolPage(props) {
                                         </div>
                                     )}
                                 </div>
-                                <div className="row justify-content-center m-0">
-                                    <ProtocolInfo title={protocol.title} description={protocol.description} />
-                                </div>
+                                {currentPageIndex === 0 && (
+                                    <div className="row justify-content-center m-0 pb-3">
```

Comment:
```suggestion
                                    <div className="d-flex flex-column justify-content-center pb-3">
```
- Não cabe row sem colunas.